dlc2nwb/utils.py

```diff
@@ -195,6 +198,7 @@ def _write_pes_to_nwbfile(
     paf_graph,
     timestamps,
     exclude_nans,
+    **optional_kwargs,
```

Comment:
```suggestion
    **optional_container_kwargs,
```

or `optional_pose_estimation_kwargs` to distinguish these from ones you would pass to individual series

Also might just be better to make it fully explicit with `pose_estimation_container_kwargs: Optional[dict] = None`

Reply:
Ok, fixed!